tests/vmcheck/test-misc-2.sh

```diff
@@ -47,7 +47,7 @@ vm_rpmostree cleanup -p
 # Add metadata string containing EnfOfLife attribtue
 META_ENDOFLIFE_MESSAGE="this is a test for metadata message"
 commit=$(vm_cmd ostree commit -b vmcheck \
-            --tree=ref=vmcheck --add-metadata-string=ostree.endoflife=\"${META_ENDOFLIFE_MESSAGE}\")
+            --tree=ref=vmcheck --add-metadata-string=ostree.endoflife="'${META_ENDOFLIFE_MESSAGE}'")
```

Comment:
This isn't strictly needed, but it was throwing off my editor syntax highlighting :rainbow: for everything else below. :) (I tend to prefer the wrapping quotes method anyway to work around the hungry SSH issue).